hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsConfigKeys.java

```diff
@@ -68,11 +68,16 @@
   public static final String HDDS_CONTAINERSCRUB_ENABLED =
       "hdds.containerscrub.enabled";
   public static final boolean HDDS_CONTAINERSCRUB_ENABLED_DEFAULT = false;
+
   public static final boolean HDDS_SCM_SAFEMODE_ENABLED_DEFAULT = true;
   public static final String HDDS_SCM_SAFEMODE_MIN_DATANODE =
       "hdds.scm.safemode.min.datanode";
   public static final int HDDS_SCM_SAFEMODE_MIN_DATANODE_DEFAULT = 1;
 
+  public static final String HDDS_CONTAINER_SCANNER_VOLUME_BYTES_PER_SECOND =
+      "hdds.container.scanner.volume.bytes.per.second";
```

Comment:
We could make this a bit more intuitive by adding let's say, throttle.bytes.per.second?

Reply:
Sorry my standard Ozone Comment: Can we please use the configuration based API for these changes.
https://cwiki.apache.org/confluence/display/HADOOP/Java-based+configuration+API

Reply:
@swagle the property name is loosely modeled after HDFS. So i think we can keep it that way.
@anuengineer thanks for the info. Let me refactor the logic here to use configuration based API.

Reply:
Updated the patch to use configuration based APIs.